block-production/Cargo.toml

```diff
@@ -33,6 +33,7 @@ nimiq-hash = { path = "../hash" }
 nimiq-keys = { path = "../keys" }
 nimiq-primitives = { path = "../primitives" }
 nimiq-transaction = { path = "../primitives/transaction" }
+nimiq-trie = { path = "../primitives/trie" }
```

Comment:
Is this right? I don't see why this is now a regular dependency

Reply:
Good catch, doesn't seem to be needed.